doc/news/changes/minor/20171028MartinKronbichler

```diff
@@ -0,0 +1,7 @@
+Fixed: The member variable FEEvaluation::dofs_per_cell and the static variable
```

Comment:
maybe should go into incompatibilities?

Reply:
I thought about that too - but then decided it equally valid to say that just fixes up what the variable name says. I have no strong opinion, so we could change it.

Reply:
i don't have a strong opinion either